test/classes/initializers/inheritance/inheritedInitWrongName.future

```diff
@@ -0,0 +1,6 @@
+error message: accidental call to superclass initializer
+
+It seems to me that calls to 'new subclass(...)' (like 'new D' in this
```

Comment:
I created issue #8176 with the intention that it would track this case - you can refer to it here if you like.